src/pyOpenMS/CMakeLists.txt

```diff
@@ -62,25 +62,35 @@ execute_process(
 message(STATUS "Python found at ${PYTHON_EXECUTABLE} with version ${PY_VERSION} (if this is wrong, configure with -DPYTHON_EXECUTABLE:FILEPATH=/path/to/python)")
 
 #------------------------------------------------------------------------------
-# Windows support restricted to Python 2.7 at the moment!
-#  * we also require VC 2008 and the corresponding vcredist to be installed
+# Windows support restricted to Python 2.[67] and Python 3.[34] at the moment!
+#  * we also require VC 2008 / 2010 and the corresponding vcredist to be installed
 if(WIN32)
-    if(NOT MSVC90)
+    if(MSVC90 AND (${PY_VERSION} STREQUAL "2.6" OR ${PY_VERSION} STREQUAL "2.7" ) )
```

Comment:
hmm not sure why this shows up here, I think this was a PR against master and the sync is just proposed?